accountserver/server.go

```diff
@@ -305,21 +305,20 @@ func (server *AccountServer) AccountDeleteHandler(writer http.ResponseWriter, re
 	_, err = db.GetInfo()
 	if err != nil {
 		srv.GetLogger(request).Error("Unable to get account info.", zap.Error(err))
-		srv.StandardResponse(writer, http.StatusInternalServerError)
+		srv.ErrorResponse(writer, err)
 		return
 	}
-	status, err := db.Delete(timestamp)
-	if err != nil {
+	if err := db.Delete(timestamp); err != nil {
```

Comment:
so the difference here is that if the delete 409s then it would rerun GetInfo like it did before- but that should be fine because the status of the DB shouldn't have changed with the 409